sdk/containerregistry/azure-containerregistry/azure/containerregistry/aio/_async_exchange_client.py

```diff
@@ -47,28 +48,37 @@ def __init__(self, endpoint, credential, **kwargs):
         if not endpoint.startswith("https://") and not endpoint.startswith("http://"):
             endpoint = "https://" + endpoint
         self._endpoint = endpoint
-        self._credential_scopes = "https://management.core.windows.net/.default"
+        self._credential_scope = "https://management.core.windows.net/.default"
         self._client = ContainerRegistry(
             credential=credential,
             url=endpoint,
             sdk_moniker=USER_AGENT,
             authentication_policy=ExchangeClientAuthenticationPolicy(),
-            credential_scopes=kwargs.pop("credential_scopes", self._credential_scopes),
+            credential_scopes=kwargs.pop("credential_scopes", self._credential_scope),
             **kwargs
         )
         self._credential = credential
+        self._refresh_token = None
+        self._refresh_expiry = None
 
     async def get_acr_access_token(self, challenge, **kwargs):
-        # type: (str) -> str
+        # type: (str, Dict[str, Any]) -> str
```

Comment:
Is there a reason to use `Dict` here for kwargs instead of `**Any`? Also, definitely Python 3 here, you could use the newer syntax.